classes/class-ed-lti.php

```diff
@@ -475,20 +464,20 @@ private function lti_redirect_user_to_blog_without_login( $blog_id ) {
 	 * @param string $alphabet
 	 *
 	 * @return string
-	 * @throws Exception
-	 * @throws InvalidArgumentException
+	 * @throws \Exception
+	 * @throws \InvalidArgumentException
 	 */
 	private function random_string( $length, $alphabet ) {
 		if ( $length < 1 ) {
-			throw new InvalidArgumentException( 'Length must be a positive integer' );
+			throw new \InvalidArgumentException( 'Length must be a positive integer' );
```

Comment:
Please change this to remove the backslash and add a `use InvalidArgumentException`